### ISSUE TITLE
Background mode

### ISSUE THREAD

**Issue by jay-cohen** [ashleymills/Reachability.swift#200]
Is it possible to use reachability in conjunction with background mode to detect a wifi connection change?

Thanks

**Comment by alexn-spire** [comment]
I would like to know this as well.  Need to detect if WiFi has been lost when the app is in background.  At the moment, we only get the notification when the app comes back into the foreground.

**Comment by Voxar** [comment]
@TheSwiftyCoder, @alexn-spire; is this even while your app is actively running in the background?

**Comment by alexn-spire** [comment]
@Voxar That is correct.  Even when the app is active in background, Reachability notifications don't fire. I still see logs/notifications for other app events while app is active in bg.
